libraries/default.rb

```diff
@@ -382,6 +382,27 @@ def create_ssl_cert(ssl_info)
         mode '0600'
       end
     end
+
+    def get_capture_agent_manager_info
```

Comment:
i went back and forth but i'm using this now (indirectly) on most capture-agent-manager recipes.